tools/c7n_gcp/c7n_gcp/client.py

```diff
@@ -188,10 +195,11 @@ def __repr__(self):
     def get_default_project(self):
         if self.project_id:
             return self.project_id
-        for k in ('GOOGLE_PROJECT', 'GCLOUD_PROJECT',
-                  'GOOGLE_CLOUD_PROJECT', 'CLOUDSDK_CORE_PROJECT'):
-            if k in os.environ:
-                return os.environ[k]
+        default_project = get_default_project()
+
```

Comment:
should probably remove the blank line to clean up the method

Reply:
updated